recipe/recipe.yaml

```diff
@@ -171,6 +171,7 @@ outputs:
         - jax >=0.7.0
         - lmfit >=1.3.4,<1.4
         - lsstdesc.coord >=1.3.0
+        - lsst-documenteer-guide
```

Comment:
do we need to remove `documenteer` itself for this to work? (at least I can't install lsst-documenteer-guide into my 11.0 env because it complains about documenteer)

I think sphinxcontrib-doxylink will need to be added in addition to this.

Reply:
I think it should work (maybe some version pin thingy) but these are redundant and `lsst-documenteer-guide` is correct https://github.com/conda-forge/lsst-documenteer-feedstock/blob/main/recipe/meta.yaml#L66-L69

Reply:
If we need `sphinxcontrib-doxylink` then that needs to be specified directly.

Reply:
Documenteer guide was fixed in https://github.com/conda-forge/lsst-documenteer-feedstock/pull/51